### DISCUSSION TITLE
No longer downloads latest driver

### DISCUSSION THREAD

**Discussion by tacticurn** [ElPumpo/TinyNvidiaUpdateChecker#106] [Q&A] [answered]
I ran TinyNvidiaUpdateChecker today and it downloaded 472.11, Installed correctly. I ran it again and it says "Your GPU drivers are up-to-date."

But latest downloadable Game Ready drivers from Nvidia's site is 497.12 from December 12. 
Studio drivers are 511.09 from January 2022
https://www.nvidia.com/en-us/drivers/results/184717/

Using Windows 10 and a GeForce 1660 Super. Is there something I could have done wrong on my end, or has Nvidia changed how they offer up drivers for download so this program cannot detect it anymore?

**Comment by HawaiiBeach** [comment]
Thanks for your question. The fact is that it's a known bug, which is that for some reason non DCH drivers no longer seem to return the latest driver version.

I've just not felt like fixing this bug due to life.. if someone could help out it'd be awesome to see some PR along here 👀 ~~Sorry the cold truth I've lost motivation to do dev things lately.~~

Happy weekend!

**Comment by HawaiiBeach** [comment, marked as answer]
Gonna finally be fixed in upcoming update
